tests/unit/ai/review/test_context_integration.py

```
@@ -0,0 +1,178 @@
+
+
+@pytest.fixture
+def git_repo(tmp_path: Path, monkeypatch: pytest.MonkeyPatch) -> GitRepo:
+    """Initialize a temporary git repository and chdir into it."""
+    git_bin = shutil.which("git")
+    if git_bin is None:
+        pytest.skip("git is required for review context integration tests")
+    if shutil.which("bash") is None:
+        pytest.skip("bash is required for git-backed review context integration tests")
+    monkeypatch.chdir(tmp_path)
+    init = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)